gcp/databases/cloudsql/private_postgres/main.tf

```diff
@@ -29,7 +29,17 @@ module cloudsql-postgres {
     ipv4_enabled = false
     private_network = data.google_compute_network.main.id
     require_ssl = true
-    allocated_ip_range = module.private-service-access.google_compute_global_address_name
+    allocated_ip_range = null
```

Comment:
Why was this changed to null?

Reply:
Had to set it to null otherwise terraform was failing with error:
"Cannot update allocated IP range: changing range within a VPC is not allowed"